core/rawdb/ancient_scheme.go

```diff
@@ -37,13 +37,18 @@ const (
 	ChainFreezerReceiptTable = "receipts"
 )
 
-// chainFreezerNoSnappy configures whether compression is disabled for the ancient-tables.
+// chainFreezerTableConfigs configures whether compression is disabled for the ancient-tables.
 // Hashes and difficulties don't compress well.
-var chainFreezerNoSnappy = map[string]bool{
-	ChainFreezerHeaderTable:  false,
-	ChainFreezerHashTable:    true,
-	ChainFreezerBodiesTable:  false,
-	ChainFreezerReceiptTable: false,
+var chainFreezerTableConfigs = map[string]freezerTableConfig{
+	ChainFreezerHeaderTable:  {snappy: false, prunable: false},
```

Comment:
the snappy options are configured incorrectly.
originally it indicates "no-compression"